schemas/schemas/adminQuery.json

```diff
@@ -8,6 +8,10 @@
         "description"
     ],
     "properties": {
+        "comment": {
+            "description": "Arbitrary comment for reference purposes.",
```

Comment:
You are probably intending to do this anyway, but this description should be used for all `"comment"` properties in all schemas.

Reply:
I'm embarrassed to say that hadn't occurred to me, but it does now... :)